lerna.json

```diff
@@ -2,5 +2,5 @@
   "packages": ["apps/*", "packages/*", "scripts", "packages/fluentui/*"],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "0.45.0"
+  "version": "0.0.0"
```

Comment:
A value there was reverted to the original one as `version` field is required in `lerna.json`